script/Arcueid/Day 7/07_00_ARC07_3_1_1.txt

```diff
@@ -895,8 +895,8 @@ One in ten thousand corpses will survive against the odds, mature,
 [sha:eab29d7480ff0e9635afe024d5a92cc704b10433]{
 -- Page 572, Offset 18008.
 -- やがて王を墜とすように――――彼らもそういうゲームでもしなければ存在してられないの。
-From there, they strive to someday dethrone the king―――
-Without engaging in such games, they wouldn't be able to go on living.
+From there, they strive to someday dethrone the king――
+without engaging in such games, they wouldn't be able to go on living.
```

Comment:
the ―― here just feels like an awkward artifact from Japanese. 
Probably a period.